chart/skywalking/templates/oap-role.yaml

```diff
@@ -13,11 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{{- if not .Values.oap.envoy.als.enabled }}
+{{- if .Values.oap.rbac.enabled }}
 kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
+  {{- if .Values.oap.rbac.useExistingRole }}
+  name: {{ .Values.oap.rbac.useExistingRole }}
+  {{- else }}
```

Comment:
Isn't this an error when you try to create another role with the same name user already defined `{{ .Values.oap.rbac.useExistingRole }}`? Also this doesn't accord to what is said in the yaml file (`skipping role creating`):

```yaml
     # Set to a rolename to use existing role - skipping role creating - but still doing serviceaccount and rolebinding to it, rolename set here.
     # useExistingRole: your-existing-role
```